spec/factories/users.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :user do
+      name {"Humlan"}
+      email {"biet@gmail.com"} 
```

Comment:
Please use standardized emails, passwords, and names in the future.  E.g `name: {'John Doe'}`, `email: {'user@mail.com'}` and `password: {'password'}`